Campus-iOS/Map/MainView/MapContent.swift

```diff
@@ -25,8 +25,10 @@ struct MapContent: UIViewRepresentable {
     @Binding var zoomOnUser: Bool
```

Comment:
Do we really need this `UIViewRepresentable` here? Shouldn't we move away from UIKit?

Reply:
I tried to use the SwiftUI map, however there were some difficulties with the showing of the pins/markers.

While it is possible to show these, they are not grouped together when they are close for example.
Also, the labels of markersmay overlap.

While it may be possible, to fix these, I felt that it would take very long, for a feature thats currently already implemented.

Reply:
K that seems like a reasonable tradeoff, lets leave it like that for now